website/docs/r/federated_settings_org_config.html.markdown

```diff
@@ -38,12 +39,22 @@ data "mongodbatlas_federated_settings_org_configs" "org_configs_ds" {
 * `post_auth_role_grants` - (Optional) List that contains the default [roles](https://www.mongodb.com/docs/atlas/reference/user-roles/#std-label-organization-roles) granted to users who authenticate through the IdP in a connected organization.
 
 * `domain_restriction_enabled` - (Required) Flag that indicates whether domain restriction is enabled for the connected organization.
-* `identity_provider_id` - (Required) Unique 24-hexadecimal digit string that identifies the federated authentication configuration.
+* `identity_provider_id` - (Required) Legacy 20-hexadecimal digit string that identifies the UI access identity provider that this connected org config is associated with. This id can be found within the Federation Management Console > Identity Providers tab by clicking the info icon in the IdP ID row of a configured identity provider.
+* `data_access_identity_provider_ids` - (Optional) The collection of unique ids representing the identity providers that can be used for data access in this organization.
```

Comment:
[q] are there examples to be updated?

Reply:
I checked out the example in [mongodbatlas_federated_settings_org_role_mapping](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/examples/mongodbatlas_federated_settings_org_role_mapping/Readme.md), and updated the default in 56b8f64

However, we will create a new example in [this Jira ticket](https://jira.mongodb.org/browse/CLOUDP-250917) that will use the new `data_access_identity_provider_ids` field 👍